docs/source/hardware/build_materials.rst

```diff
@@ -35,17 +35,17 @@ The following parts are required per quadrotor:
 ================================================================ === ================================================================================================================================
 Components                                                       Qty           Link
 ---------------------------------------------------------------- --- --------------------------------------------------------------------------------------------------------------------------------
-PixFalcon Autopilot w/ GPS and Power Module                       1        `HobbyKing <https://hobbyking.com/en_us/pixfalcon-micro-px4-autopilot-plus-micro-m8n-gps-and-mega-pbd-power-module.html>`_
-F330 Frame                                                        1        `Amazon <https://www.amazon.com/gp/product/B00S9BP5MU>`_
-Motors: Crazepony 4pcs Emax MT2213 935KV                          1        `Amazon <https://www.amazon.com/gp/product/B00XY6T1W0>`_
-8045 Propeller Set (2xCW, 2xCCW>)                                 1        `Amazon <https://www.amazon.com/gp/product/B00T3VR85W/>`_
+PixFalcon Autopilot w/ GPS and Power Module                       1        `Amazon <https://www.amazon.com/Pixfalcon-M8N-GPS-Advanced-Bundle/dp/B017YCYVRO>`_
+F330 Frame                                                        1        `Amazon <https://www.amazon.com/Quadcopter-Frame-Aircraft-Accessory-Integrated/dp/B07D6K51DY>`_
+Motors: Crazepony 4pcs Emax MT2213 935KV                          1        `Amazon <https://www.amazon.com/MT2213-935KV-Brushless-Motor-Quadcopter-Multirotor/dp/B00N3I9GM4>`_
+8045 Propeller Set (2xCW, 2xCCW>)                                 1        `Amazon <https://www.amazon.com/DJI-Innovations-Quadcopter-Original-Propellers/dp/B00BS40S4A>`_
 ESC: ZTW Spider 30A OPTO 2-6S LiPo 600HZ Simonk OneShot125 ESC    4        `Amazon <https://www.amazon.com/gp/product/B01BXS7NQ6>`_
 Spektrum 9645 DSMX Remote Receiver                                1        `Amazon <https://www.amazon.com/gp/product/B004M12GY6/>`_
```

Comment:
this one is out of stock

Reply:
Looks like ESC, Receiver, and Transmitter are out-of-stock on many e-commerce sites. Leaving as they are based on our discussion last Monday.